2021-08-31 18:46:13,720 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-08-31 18:46:13,720 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /usr/local/lib/python3.9/site-packages/instabot/bot/bot.py
2021-08-31 18:46:13,721 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-08-31 18:46:14,283 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2021-08-31 18:46:14,638 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2021-08-31 18:46:14,884 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2021-08-31 18:46:15,101 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2021-08-31 18:46:17,322 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [200]>
2021-08-31 18:46:17,322 - instabot version: 0.117.0 (api) - INFO - Logged-in successfully as 'goga_osoka'!
2021-08-31 18:46:17,322 - instabot version: 0.117.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: True
2021-08-31 18:46:17,559 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [400]>
2021-08-31 18:46:17,560 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/qe/sync/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:46:17,560 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:46:17,560 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:46:17,560 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:46:17,771 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [400]>
2021-08-31 18:46:17,771 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/launcher/sync/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:46:17,771 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:46:17,771 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:46:17,771 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:46:17,998 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: zr/token/result/?device_id=android-d0b0f4c504afc855&token_hash=&custom_device_id=android-d0b0f4c504afc855&fetch_reason=token_expired returned response: <Response [400]>
2021-08-31 18:46:17,998 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/zr/token/result/%253Fdevice_id%253Dandroid-d0b0f4c504afc855%2526token_hash%253D%2526custom_device_id%253Dandroid-d0b0f4c504afc855%2526fetch_reason%253Dtoken_expired","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:46:17,998 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:46:17,998 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:46:17,998 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:46:18,198 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: multiple_accounts/get_account_family/ returned response: <Response [400]>
2021-08-31 18:46:18,198 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/multiple_accounts/get_account_family/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:46:18,198 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:46:18,198 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:46:18,199 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:46:18,415 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [400]>
2021-08-31 18:46:18,416 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/qe/sync/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:46:18,416 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:46:18,416 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:46:18,416 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:46:18,630 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: igtv/browse_feed/?prefetch=1 returned response: <Response [400]>
2021-08-31 18:46:18,630 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/igtv/browse_feed/%253Fprefetch%253D1","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:46:18,630 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:46:18,630 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:46:18,630 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2021-08-31 18:46:18,967 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: creatives/ar_class/ returned response: <Response [400]>
2021-08-31 18:46:18,967 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/?next=/api/v1/creatives/ar_class/","api_path":"/challenge/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail"}'
2021-08-31 18:46:18,968 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2021-08-31 18:46:18,968 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2021-08-31 18:46:18,968 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
